internal/fnruntime/runner.go

```diff
@@ -179,32 +179,35 @@ type FunctionRunner struct {
 
 func (fr *FunctionRunner) Filter(input []*yaml.RNode) (output []*yaml.RNode, err error) {
 	pr := printer.FromContextOrDie(fr.ctx)
```

Comment:
Maybe not for this PR, but we need to remove use of the printer from all of kpt that isn't the cli commands.